benches/bench.rs

```diff
@@ -132,13 +132,34 @@ fn bench_get_many_large(c: &mut Criterion) {
     });
 }
 
+fn bench_increment(c: &mut Criterion) {
+    let rt = Runtime::new().unwrap();
+
+    rt.block_on(async {
```

Comment:
This seems like a bug; we're setting "foo" to "bar" and then later on calling increment on "foo". Shouldn't the initial `set` call instead be: `client.set("foo", 0, None, None).await.unwrap();`